docroot/themes/humsci/humsci_basic/theme-settings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Provides an additional config form for theme settings.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\Core\Link;
+use Drupal\Core\Render\Markup;
+
+
+// Set theme name to use in the key values.
+$theme_name = \Drupal::theme()->getActiveTheme()->getName();
+
+/**
+ * Implements hook_form_system_theme_settings_alter().
+ *
+ * Form override for theme settings.
+ */
+function humsci_basic_form_system_theme_settings_alter(array &$form, FormStateInterface $form_state) {
+  $form['options_settings'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Theme Specific Settings'),
+  ];
+
+  // Global Footer.
+  $form['options_settings']['humsci_basic_global_footer'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Global Footer Settings'),
+  ];
+
+  $form['options_settings']['humsci_basic_global_footer']['global_footer_variant'] = [
+    '#type' => 'select',
+    '#title' => t('Global Footer Variant'),
+    '#options' => [
+      'default' => '- Default -',
+      'bright' => t('Bright'),
+      'dark' => t('Dark')
+    ],
+    '#default_value' => theme_get_setting('global_footer_variant'),
+  ];
+
+  // BrowserSync support.
```

Comment:
do we want this browser sync support? sounds intriguing

Reply:
Yeah. I wasn't sure. I don't think we want to take out something they may be expecting to be there. I saw your note about it in Slack so we'll see. :smile:

Reply:
yeah @heatherdesigns I don't think it hurts to leave in for now :)

Reply:
I've created a separate Jira card for adding Browsersync settings. This will allow the initiation of a conversion with H&S to see whether or not it is a setting they want to be included. See [JIRA STN-189](https://sparkbox.atlassian.net/browse/STN-189)

Reply:
Thanks @heatherdesigns :)